Build/NuSpec/Web.targets

```diff
@@ -9,16 +9,40 @@
   </ItemGroup>
 
   <PropertyGroup>
+    <!-- Old .NET Framwork ASP.NET projects are detected using the project type guid -->
     <WebApplicationProjectTypeGuid>{349c5851-65df-11da-9384-00065b846f21}</WebApplicationProjectTypeGuid>
-    <RemotionWebResource_IsWebApplication>$(ProjectTypeGuids.ToLower().Contains($(WebApplicationProjectTypeGuid)))</RemotionWebResource_IsWebApplication>
+    <_isNetFramworkWebApplicationProject>$(ProjectTypeGuids.ToLower().Contains($(WebApplicationProjectTypeGuid)))</_isNetFramworkWebApplicationProject>
+    <!-- .Contains returns 'True'/'False' so we need to normalize to lowercase here -->
+    <_isNetFramworkWebApplicationProject>$(_isNetFramworkWebApplicationProject.ToLower())</_isNetFramworkWebApplicationProject>
```

Comment:
Wouldn't it be cleaner to do this all in one?

Reply:
Problem is that I could not get the conversion from `bool` to `string` to work as `.ToLower` only works on string. This way, the conversion is done implicitly.

Reply:
Ah, yes. Personally, I probably would go with `.ToStirng().ToLower()` in that case. As you like?